docs/apache-airflow-providers-google/sensors/cloud/gcs.rst

```diff
@@ -0,0 +1,75 @@
+ .. Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+ ..   http://www.apache.org/licenses/LICENSE-2.0
+
+ .. Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+
+
+Google Cloud Storage Sensors
+============================
+
+Cloud Storage allows world-wide storage and retrieval of any amount of data at any time. 
+You can use Cloud Storage for a range of scenarios including serving website content, 
+storing data for archival and disaster recovery, or distributing large data objects to users via direct download.
+
+.. contents::
+  :depth: 1
+  :local:
+
+Prerequisite Tasks
```

Comment:
The two headings you add with the two example sensors show up under Prerequisite tasks - is that where you want them? Or is it that prerequisite tasks is empty and it's rendering incorrectly?

Reply:
Oops I hadn't seen the other prerequisites file - I think it's not rendering properly

Reply:
Cheers - I think it should be fine now